distributed/http/worker/prometheus/core.py

```diff
@@ -26,15 +28,15 @@ def __init__(self, server: Worker):
     def collect(self):
         from prometheus_client.core import CounterMetricFamily, GaugeMetricFamily
 
+        ws = self.server.state
+
         tasks = GaugeMetricFamily(
             self.build_name("tasks"),
             "Number of tasks at worker.",
             labels=["state"],
         )
-        tasks.add_metric(["stored"], len(self.server.data))
-        tasks.add_metric(["executing"], self.server.state.executing_count)
-        tasks.add_metric(["ready"], len(self.server.state.ready))
-        tasks.add_metric(["waiting"], self.server.state.waiting_for_data_count)
+        for k, n in ws.task_counts.items():
+            tasks.add_metric([k], n)
```

Comment:
Renamed stored -> memory

Reply:
what's the difference between things in `data` and actors in "memory" state? (I noticed that actors weren't included in previous metric and are included in new metric)

Reply:
The "real" instance of an actor lives in WorkerState.actors instead of data. Its proxies on remote workers live in data. Actors are an edge case I would not spend time fine-tuning anything towards anyway.